flink-core/src/main/java/org/apache/flink/api/common/state/AppendingState.java

```diff
@@ -29,7 +29,7 @@
  * <p>The state is accessed and modified by user functions, and checkpointed consistently
  * by the system as part of the distributed snapshots.
  * 
- * <p>The state is only accessible by functions applied on a KeyedDataStream. The key is
+ * <p>The state is only accessible by functions applied on a KeyedStream. The key is
```

Comment:
I probably not set this to ```@link``` , instead, I would set this to ```@code```, because if I use ```link```, it can not be recognized by compiler. I have to add the pom.xml to the module. And I think it is not suitable. What do you think ?